src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -145,7 +145,10 @@ public static void InitializeForTesting(List<TypeInfo> serializationProviders =
                 BufferPool.InitGlobalBufferPool(new MessagingConfiguration(false));
                 RegisterSerializationProviders(serializationProviders);
                 AssemblyProcessor.Initialize();
+#if !NETSTANDARD_TODO
+                 //blocked by Cannot use binary formatter as fallback serializer while running on .Net Core
                 fallbackSerializer = GetFallbackSerializer();
+#endif
```

Comment:
there's already an #if in the GetFallbackSerializer method itself, please change that to return null if that's what you want to do termporarily.